File: back-end/member-service/src/main/java/com/muates/memberservice/model/dto/response/MemberResponse.java
package com.muates.memberservice.model.dto.response;

import com.muates.memberservice.model.enums.Gender;
import lombok.Builder;
import lombok.Data;

import java.util.Date;

@Data
@Builder
public class MemberResponse {
    private Long id;
    private String username;
    private String email;
    private String firstName;
    private String lastName;
    private Gender gender;
    private Date birthDate;
    private String country;
    private String profileImageUrl;
    private String biography;
    private String website;
    private String links;
    private String socialMediaLinks;
    private Date createdDate;
    private Date updatedDate;
}

File: front-end/src/app/core/models/member/response/member-response.ts
export class MemberResponse {
    id!: number;
    username!: string;
    email!: string;
    firstName!: string;
    lastName!: string;
    gender!: 'MALE' | 'FEMALE' | 'OTHER';
    birthDate!: Date;
    country!: string;
    profileImageUrl!: string;
    biography!: string;
    website!: string;
    links!: string;
    socialMediaLinks!: string;
    createdDate!: Date;
    updatedDate!: Date;
}
